src/pages/gd/Stake/index.tsx

```diff
@@ -202,14 +205,17 @@ const StakeTable = ({
                                                 setActiveStake(stake)
                                                 setActiveTableName()
                                             }}
-                                        >
-                                            {i18n._(t`Stake`)}
+                                        > { stake.isV2 || stake.protocol === LIQUIDITY_PROTOCOL.GOODDAO ?
```

Comment:
this should be same condition as the "disabled" on line 198. just so it is more clear